docs/developer-guides/setting-up-token-management-partner-chains.md

```diff
@@ -0,0 +1,402 @@
+
+## 1.3 Initializing token reserve controls
+
+Objective: Initialize the reserve management system for your token.
+
+### 1.3.1 Run `init-reserve-management` command
+
+Command template:
+
+```bash
+./pc-contracts-cli init-reserve-management \
+--genesis-utxo <GenesisCommitteeHashUTxO> \
+--ogmios-host <Ogmios_Host> \
+--ogmios-port <Ogmios_Port> \
+--kupo-host <Kupo_Host> \
+--kupo-port <Kupo_Port> \
+--payment-signing-key-file <PaymentSigningKeyFile> \
+--network testnet
+```
+
+### Steps
+
+1. Execute the `init-reserve-management` command.
+
+2. Confirm initialization: Ensure that the `InitReserveManagement` scripts are correctly initialized by checking the command output.
+
+## 1.4 Creating your token reserve
+
+Objective: Create a reserve for your token and define the release function `V`.
+
+### 1.4.1 Run `reserve-create` command
+
+Command template:
+
+```bash
+/pc-contracts-cli reserve-create \
+  --genesis-utxo <GenesisCommitteeHashUTxO> \
+  --total-accrued-function-script-hash <VFunctionScriptHash> \
+  --reserve-posixtime-t0 <POSIXTimeT0> \
+  --reserve-asset-script-hash <ReserveAssetScriptHash> \
+  --reserve-asset-name <ReserveAssetName> \
+  --reserve-initial-deposit-amount <InitialDepositAmount> \
+  --ogmios-host <Ogmios_Host> \
+  --ogmios-port <Ogmios_Port> \
+  --kupo-host <Kupo_Host> \
+  --kupo-port <Kupo_Port> \
+  --payment-signing-key-file <PaymentSigningKeyFile> \
+  --network <Network>
+```
+
+#### Example command
+
+```bash
+/pc-contracts-cli reserve-create \
+  --genesis-utxo b88dd63b770357e58c30d17f364a7b681a8fd52f721614db35977594 \
+  --total-accrued-function-script-hash e833119231cabfc4de0733b05e0889a736373079c414a1b8c9ae6ac0 \
+  --reserve-posixtime-t0 1729851426 \
+  --reserve-asset-script-hash 55e8a5e0df12e794f2e78f55848c7cbd8f6de9a23be16be98d769116 \
+  --reserve-asset-name 416c657854657374546f6b656e \
+  --reserve-initial-deposit-amount 1000000 \
+  --payment-signing-key-file payment.skey \
+  --ogmios-host localhost \
+  --ogmios-port 1337 \
+  --kupo-host localhost \
+  --kupo-port 1442 \
+  --network testnet
+```
+
+#### Explanation of parameters
+
+* `--genesis-utxo`: The genesis UTXO of the running partnerchain.
+
+* `--network`: The network you operate on (`testnet` or `mainnet`).
+
+* `--total-accrued-function-script-hash`: The script hash of your `V` function.
+
+* `--reserve-posixtime-t0`: an integer that could possibly be used by user defined `V` function. It will be written into PlutusData of UTXO at Reserve Validator. Deprecated. This field will be set to 0 in the future versions of CLI.
+
+* `--reserve-asset-script-hash`: The script hash of the reserve asset (policy ID).
+
+* `--reserve-asset-name`: The asset name in hexadecimal.
+
+* `--reserve-initial-deposit-amount`: The initial amount of tokens to deposit into the reserve.
+
+* `--payment-signing-key-file`: Your payment signing key file for transaction signing.
```

Comment:
It is worth noting that this key has to have `reserve-initial-deposit-amount` in it's wallet.

Reply:
I've added that info to this statement: 

* `--payment-signing-key-file`: Your payment signing key file for transaction signing. This key has to have `reserve-initial-deposit-amount` in its wallet.